src/components/Toggle/README.md

```diff
@@ -0,0 +1,142 @@
+---
+name: Toggle
+category: Forms
+platforms:
+  - web
+keywords:
+  - toggle
+  - switch
+  - selection
+  - choices
+  - options
+  - pick
+  - single selection form
+  - choice form
+  - option button
+  - enable
+  - disable
+---
+
+# Toggle
+
+Use to present a single choice to enable or disable a setting.
+
+---
+
+## Best practices
+
+Toggle should:
+
+- Always be used with an associated label component.
+- Be used to switch a single setting on and off.
```

Comment:
It would be good to help developers figure how to choose between a toggle and a checkbox. What should drive that decision? Is it the surrounding context, the space available, the size of the label,..?